lib/microform/test_methods.rb

```diff
@@ -0,0 +1,39 @@
+require "minitest/stub_any_instance"
+require "minitest/mock"
+
+module Microform
+  module TestMethods
+    def assert_submits form_kind, stub: nil, &block
+      controller = self.class.name.gsub(/Test\z/, "").constantize
+      form = form_kind.new OpenStruct.new(foo: :bar)
+
+      __microform_form_kind = form_kind
```

Comment:
What are these leading underscores meant to represent? To highlight that they are in the stubbed method? Or something else? I read them as "private" from C#, at first glance.